cmd/gen-settings/main.go

```
@@ -0,0 +1,330 @@
+		Category:    {{.Category}},
+	},
+{{- end}}
+}
+`))
+
+func writeOutput(path string, settings []SettingDef) error {
+	f, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)